screens/pages/components/albums/src/index.js

```diff
@@ -129,12 +133,13 @@ class GalleryMediaPicker extends Component {
                 ? 0
                 : this.props.gallery[galleryLength - 1].id + 1;
 
-        const geotagged = items.map(item => {
+        items.map(item => {
```

Comment:
I think we should be using `.filter` here instead of pushing into a new array?

Reply:
this file is not used anymore, this is part of the code forked from library..
Now I moved this code to gallery_action
Actually filter Creates a New Array too and have a subset of the original array that passes the condition.
But here we are changing/ modifying the shape of the object in array , so in that case we have to use map